migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,68 @@
+
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[Install migration Toolkit for Containers Operator on {product-title}]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#configuring-replication-repository_installing-mtc[Configure replication repository]
+
+[discrete]
+[id="mtc-overview-upgrade-mtc"]
+== Upgrading MTC
+You can xref:../migration_toolkit_for_containers/upgrading-mtc.adoc#upgrading-mtc[upgrade the MTC] by using Operator Lifecycle Manager.
+
+[discrete]
+[id="mtc-overview-mtc-checklists"]
+== Reviewing MTC checklists
+Before you migrate your application workloads with the Migration Toolkit for Containers (MTC), you can review the xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#premigration-checklists-mtc[premigration checklists].
+
+[discrete]
+[id="mtc-overview-migrate-mtc-applications"]
+== Migrating applications
+You can migrate your applications by using the Migration Toolkit for Containers (MTC) xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migrating-applications-mtc-web-console_migrating-applications-with-mtc[web console] or xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migrating-applications-cli_advanced-migration-options-mtc[the command line].
+
+[discrete]
+[id="mtc-overview-advanced-migration-options"]
+== Reviewing advanced migration options
+You can automate your migrations and modify the `MigPlan` and `MigrationController` custom resources in order to perform large-scale migrations and to improve performance. You can check the following items:
+
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-creating-registry-route-for-dim_advanced-migration-options-mtc[Create a registry route for direct image migration]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-configuring-proxies_advanced-migration-options-mtc[Configuring proxies]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-migrating-applications-api_advanced-migration-options-mtc[Migrate an application by using the MTC API]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-state-migration-cli_advanced-migration-options-mtc[Running a state migration]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-hooks_advanced-migration-options-mtc[Creating migration hooks]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-plan-options_advanced-migration-options-mtc[Options for migrating resources]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-controller-options_advanced-migration-options-mtc[Migration controller options for improving performance]
+
+[discrete]
+[id="mtc-overview-troubleshooting-mtc"]
+== Troubleshooting MTC
```

Comment:
Troubleshooting MTC > Troubleshooting or Troubleshooting migrations

You're not troubleshooting MTC. You're troubleshooting failed or partially failed migrations.